Easybuoy/pstk-swapi-challenge
# Setup and Configure Redux

#### What does this PR do?
- Setup and Configure Redux

#### Description of Task to be completed?
- Install `redux`, `react-redux`, `redux-thunk` packages with `npm`
- Create `actions` directory and setup redux actions.
- Create `reducers` directory and setup reducers.
- Create `store.js` file and add store configurations.

#### How should this be manually tested?
- Install `redux-dev-tools` extension on your browser.
- Open `redux-dev-tools` on your browser to see the application state.

#### What are the relevant pivotal tracker stories?
- #169632820

#### Screenshots (if appropriate)
- None
